misc/images/ubuntu-base/Dockerfile

```diff
@@ -19,3 +19,7 @@ ENV RUST_LIB_BACKTRACE=0
 RUN sed -i -e 's#http://archive\.ubuntu\.com#http://us-east-1.ec2.archive.ubuntu.com#' \
            -e 's#http://security\.ubuntu\.com#http://us-east-1.ec2.archive.ubuntu.com#' \
            -e 's#http://ports\.ubuntu\.com#http://us-east-1.ec2.ports.ubuntu.com#' /etc/apt/sources.list.d/ubuntu.sources
+
+RUN apt-get update --fix-missing && TZ=UTC DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends eatmydata
+# This is unset in entrypoint.sh again if $MZ_EAT_MY_DATA is not set
+ENV LD_PRELOAD=libeatmydata.so
```

Comment:
This causes me some amount of stress if we ever accidentally use this image in something that is not supposed to lose data, and we'll never catch it in CI. Would it be better if this env var is specified in the service definitions in our python code?